fuse_doc/doc/getting_started.rst

```diff
@@ -0,0 +1,253 @@
+
+.. code-block:: yaml
+
+  motion_models:
+    unicycle_motion_model:
+      type: fuse_models::Unicycle2D
+
+  unicycle_motion_model:
+    #                         x      y      yaw    vx     vy     vyaw   ax   ay
+    process_noise_diagonal: [0.100, 0.100, 0.100, 0.100, 0.100, 0.100, 0.1, 0.1]
+    scale_process_noise: true
+
+This section specifies the motion (kinematic) model that we will use in this problem. As our robot is a differential-drive bot, we use a 2D unicycle model. Note that `fuse` supports multiple motion models to be used, but most applications will only require one.
+
+The motion model will be used to add constraints to the graph between sensor measurements. The model we have specified is of type `fuse_models::Unicycle2D`, which is a plugin with its own parameters. Those parameters are specified in the next block.
+
+The `process_noise_diagonal` specifies the error growth for each of our state variables when we apply the kinematic model. This is equivalent to the process noise covariance you might see in an EKF application. Here, we just specify the diagonals for that matrix. Additionally, the process noise can be scaled depending on how much motion was actually measured. This prevents the state estimate covariance from growing when, for example, the robot is stationary.
+
+.. code-block:: yaml
+
+  sensor_models:
+    initial_localization_sensor:
+      type: fuse_models::Unicycle2DIgnition
+      motion_models: [unicycle_motion_model]
+      ignition: true
+    odometry_sensor:
+      type: fuse_models::Odometry2D
+      motion_models: [unicycle_motion_model]
+
+  initial_localization_sensor:
+    publish_on_startup: true
+    #                x      y      yaw    vx     vy     vyaw    ax     ay
+    initial_state: [0.000, 0.000, 0.000, 0.000, 0.000, 0.000, 0.000, 0.000]
+    initial_sigma: [0.100, 0.100, 0.100, 0.100, 0.100, 0.100, 0.100, 0.100]
+
+  odometry_sensor:
+    topic: "odom"
+    twist_target_frame: "base_footprint"
+    linear_velocity_dimensions: ['x', 'y']
+    angular_velocity_dimensions: ['yaw']
+
+In this section, we specify two sensor models.
+
+The first is an "ignition" model of type `fuse_models::Unicycle2DIgnition`. It is responsible for adding a constraint to our graph for the robot's initial pose.
+  - The `publish_on_startup` parameter will cause it to add a constraint to the graph and soon as it initializes
+  - The `initial_state` and `initial_sigma` provide the starting state and covariance diagonal values
+
+The second sensor model is of type `fuse_models::Odometry2D`. This particular sensor model takes in ROS `nav_msgs/Odometry` messages and creates graph constraints from them.
+  - The `topic` parameter is the ROS topic on which to listen for ROS `nav_msgs/Odometry` messages.
+  - The `twist_target_frame` is the frame into which we want to transform the twist (velocity) data in the incoming message. In this case, we want to transform it into the *base_link* frame.
```

Comment:
```suggestion
  - The `twist_target_frame` is the frame into which we want to transform the twist (velocity) data in the incoming message. In this case, we want to transform it into the *base_footprint* frame.
```
Or otherwise change the configuration to use `base_link`

Reply:
No, `base_footprint` is right, thanks. I have a bag file that will accompany this (I'll update the tutorial with a download link).